ui/app/AppLayouts/Wallet/popups/swap/SwapInputParamsForm.qml

```diff
@@ -45,12 +47,9 @@ QtObject {
         return !!root.selectedAccountAddress &&
                 root.selectedNetworkChainId !== -1 &&
                 !!root.fromTokensKey && !!root.toTokenKey &&
-                ((!!root.fromTokenAmount &&
-                  !isNaN(SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount)) &&
-                  SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount) > 0) ||
-                 (!!root.toTokenAmount &&
-                  !isNaN(SQUtils.AmountsArithmetic.fromString(root.toTokenAmount)) &&
-                  SQUtils.AmountsArithmetic.fromString(root.toTokenAmount) > 0 )) &&
+                (!!root.fromTokenAmount &&
+                 !isNaN(SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount)) &&
+                 SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount) > 0) &&
```

Comment:
Similar here:
```suggestion
                 SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount).gt(0) 
 &&
```

Reply:
I think there is some mistake in what you typed?

Reply:
You are comparing a Big number with a JS Number, not sure it'd work in all cases; the proposed code fixes that

Reply:
using AmountsArithmetic's cmp function instead now.